src/components/button/index.tsx

```diff
@@ -440,6 +460,14 @@ function createStyles() {
       backgroundColor: 'transparent',
       flexDirection: 'row',
       ...Typography.text70
+    },
+    backgroundElement: {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      right: 0,
+      bottom: 0,
```

Comment:
If you use our `View`, you can use the `absF` modifier to avoid these (except the `overflow`).
Up to you.